runtime/gc_include/ObjectAccessBarrierAPI.hpp

```diff
@@ -2789,27 +2789,33 @@ class MM_ObjectAccessBarrierAPI
 	/**
 	 * Check to see if the object is marked
 	 *
-	 * @NOTE only for WRT as this check the mark map for segregated heaps
+	 * @NOTE only for SATB Concurrent Mark
```

Comment:
we need to remove (from here and from call sites, too):
#if defined(J9VM_GC_REALTIME)

(we need to think if this we need a new build flag or not)

Reply:
I was thinking to leave the build flags until we decide on new/rename build flag. Leaving the flags will help in introducing  (or renaming) the new flags. There are many spots that need to be looked at for the flag, so it should done as part of separate change.

Reply:
For now, standard SATB requires  the`J9VM_GC_REALTIME` build flag

Reply:
sounds good, let's leave it for now